business_requirement_deliverable_crm/wizard/crm_make_sale.py

```diff
@@ -0,0 +1,114 @@
+# © 2016-2019 Elico Corp (https://www.elico-corp.com).
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+from odoo import api, fields, models
+from odoo.tools.translate import _
+from odoo.exceptions import ValidationError
+
+
+class CrmMakeSale(models.TransientModel):
+    _name = "br.crm.lead"
+
+    @api.multi
+    def default_partner(self):
+        context = self.env.context
+        case_id = context and context.get('active_ids', []) or []
+        case_id = case_id and case_id[0] or False
+        crm_id = self.env['crm.lead'].browse(case_id)
+        return crm_id and crm_id.partner_id.id or ''
+
+    partner_id = fields.Many2one(
+        'res.partner',
+        'Customer',
+        default=default_partner
+    )
+    update_quotation = fields.Boolean('Update existing quotation')
+
+    @api.multi
+    def make_order(self):
```

Comment:
You might need to refactor this one to be able to fit to #300 
We should be able to generate a SQ either from Opp (based onlist of BR in the linked project) or based on the list of selected BR(s)